bundle/manifests/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -13,14 +13,13 @@ metadata:
           "spec": {
             "gitHubSecretName": "github-api-token",
             "kubeconfigSecretName": "kubeconfig",
-            "openShiftPipelineVersion": "v1.5.0",
             "operatorPipelinesRelease": "main",
             "pyxisSecretName": "pyxis-api-secret"
           }
         }
       ]
     capabilities: Basic Install
-    operatorframework.io/suggested-namespace: oco
+    operatorframework.io/suggested-namespace: openshift-operators
```

Comment:
I think this will solve all our weird issues. Rebuilding my bundle/catalog/etc now to test. 🤞🏻

Reply:
This now works!!!!!!!!!!!!!!!